demo_nodes_cpp/src/parameters/ros2param.cpp

```diff
@@ -139,6 +139,12 @@ int main(int argc, char ** argv)
       fprintf(stderr, "Failed to set parameter\n");
       return 1;
     }
+    auto results = set_parameters_result.get();
+    for (auto result : results) {
+      if (result.successful != true) {
```

Comment:
I was looking at https://github.com/rohbotics/demos/blob/948ca1ef6b228e7aea94eff2b6b761ea0b2c2bfc/demo_nodes_cpp/src/parameters/ros2param.cpp#L138 

And https://github.com/rohbotics/demos/blob/948ca1ef6b228e7aea94eff2b6b761ea0b2c2bfc/demo_nodes_cpp/src/parameters/ros2param.cpp#L113

But those aren't comparing to `true`, to I changed this line.